template/freemarker/src/main/java/com/blazebit/notify/template/freemarker/FreemarkerTemplateProcessor.java

```diff
@@ -78,7 +78,7 @@ public class FreemarkerTemplateProcessor implements TemplateProcessor<String>, S
     public FreemarkerTemplateProcessor(ConfigurationSource configurationSource) {
         Function<String, FreemarkerTemplateLookup> templateAccessor = name -> (Locale locale) -> {
             Configuration configuration = configurationSource.getPropertyOrDefault(FREEMARKER_CONFIGURATION_PROPERTY, Configuration.class, null, o -> {
-                Configuration c = new Configuration();
+                Configuration c = new Configuration(Configuration.VERSION_2_3_28);
```

Comment:
Why is this necessary? I'd like to avoid having to update this line of code for every bug fix update of Freemarker.

Reply:
As mentioned in the commit message, the no-param constructor is deprecated. The only alternative is the constructor that accepts the incompatibleImprovements parameter. However, the feemarker configuration can be provided by the user anyways so not sure if this is such a big issue.

Reply:
Is there no static function `Configuration.getLatestVersion()` or something like that?

Reply:
There is a `public static Version getVersion()` but the javadoc specifically says
> do NOT use this as the value of the incompatible_improvements setting (as the parameter to Configuration(Version)), as then your application can break when you upgrade FreeMarker! Use a constant value, like VERSION_2_3_28, to protect your application from fixes/changes that aren't entirely backward compatible.

Reply:
Hmm, ok. I don't think this is good design from Freemarker, but whatever, let's use the static version then.